### ISSUE TITLE
OpenVPN/easy-rsa Github Link 404

### ISSUE THREAD

**Issue by colinbs** [pivpn/pivpn#715]
# PiVPN Issue Template

## Issue
The URL that points to the `easy-rsa` tar does not seem to exist anymore:
```bash
wget -O - "https://github.com/OpenVPN/easy-rsa/releases/download/v3.0.6/EasyRSA-3.0.6.tgz"

48:51--  https://github.com/OpenVPN/easy-rsa/releases/download/v3.0.6/EasyRSA-3.0.6.tgz
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-04-10 17:48:52 ERROR 404: Not Found.
```
Prior to this line in the install script, the `/etc/openvpn/easy-rsa` directory is removed.

This results in the easy-rsa directory not existing which, again, leads to a whole lot of other problems.

**Comment by ant0nisk** [comment]
As I mentioned [here](https://github.com/pivpn/pivpn/issues/713#issuecomment-481744434), the URL is indeed wrong. 

You can download the installation file: `curl -L https://install.pivpn.io > install.sh`
Edit it with `nano` or your preferred text editor and change the following line (top of the file):
`easyrsaRel="https://github.com/OpenVPN/easy-rsa/releases/download/v${easyrsaVer}/EasyRSA-${easyrsaVer}.tgz"`

The correct variable should be this one:
`easyrsaRel="https://github.com/OpenVPN/easy-rsa/releases/download/v${easyrsaVer}/EasyRSA-unix-v${easyrsaVer}.tgz"`

So if you change this on the top of the installation script and run it (`sudo bash install.sh`), then it should work out fine.

**Comment by colinbs** [comment]
Alright, thanks for the quick answer!

**Comment by Giraffe1966** [comment]
The easy-rsa problem has been fixed (#716)
